i18n-helpers/src/bin/mdbook-xgettext.rs

```diff
@@ -31,7 +31,7 @@ use std::{fs, io};
 
 fn add_message(catalog: &mut Catalog, msgid: &str, source: &str) {
     let sources = match catalog.find_message(None, msgid, None) {
-        Some(msg) => format!("{}\n{}", msg.source(), source),
+        Some(msg) => format!("{}\n{source}", msg.source()),
```

Comment:
In cases like this where not everything can go inline, I think it's probably more readable to keep all format args consistently out of line. Having them out of order is confusing.

Reply:
This got reverted.